Styles/components.css

```diff
@@ -1,38 +1,30 @@
 @import url("https://fonts.googleapis.com/css2?family=Rubik:wght@300&display=swap");
 
```

Comment:
Having separate CSS files for different components will be easy to debug and also while reusing it. Try to separate CSS for components and import it in the main components file.